### ISSUE TITLE
App is crashing on start with latest 2.2.16 release, only at Android, only in Release mode

### ISSUE THREAD

**Issue by Szymaniuk** [luberda-molinet/FFImageLoading#740]
I've updated today FFImageLoading plugin for my Xamarin Forms app (that uses latest 2.4.0.275-pre3 release), and it's working fine in DEBUG on Android and iOS, but in RELEASE mode, it's working only on iOS - Android app is crashing during XF initialization.
Below stacktrace:

```
09-27 09:38:57.424 21876 21876 I MonoDroid: UNHANDLED EXCEPTION:
09-27 09:38:57.458 21876 21876 I MonoDroid: System.NullReferenceException: Object reference not set to an instance of an object
09-27 09:38:57.458 21876 21876 I MonoDroid:   at FFImageLoading.Forms.Droid.CachedImageRenderer.OnElementChanged (Xamarin.Forms.Platform.Android.ElementChangedEventArgs`1[TElement] e) [0x000dd] in <f6f23c6d2c6249e58850da13df5ea928>:0
09-27 09:38:57.458 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].SetElement (TElement element) [0x000b9] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.458 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].Xamarin.Forms.Platform.Android.IVisualElementRenderer.SetElement (Xamarin.Forms.VisualElement element) [0x00027] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.458 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.Platform.CreateRenderer (Xamarin.Forms.VisualElement element) [0x0001f] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.458 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.VisualElementPackager.AddChild (Xamarin.Forms.VisualElement view, Xamarin.Forms.Platform.Android.IVisualElementRenderer oldRenderer, Xamarin.Forms.Platform.Android.RendererPool pool, System.Boolean sameChildren) [0x00023] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.458 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.VisualElementPackager.SetElement (Xamarin.Forms.VisualElement oldElement, Xamarin.Forms.VisualElement newElement) [0x0010d] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.458 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.VisualElementPackager.Load () [0x0000d] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.458 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].SetPackager (Xamarin.Forms.Platform.Android.VisualElementPackager packager) [0x00007] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.458 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].SetElement (TElement element) [0x000d5] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.458 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].Xamarin.Forms.Platform.Android.IVisualElementRenderer.SetElement (Xamarin.Forms.VisualElement element) [0x00027] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.458 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.Platform.CreateRenderer (Xamarin.Forms.VisualElement element) [0x0001f] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.458 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.ScrollViewContainer.set_ChildView (Xamarin.Forms.View value) [0x0002f] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.458 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.ScrollViewRenderer.LoadContent () [0x00011] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.458 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.ScrollViewRenderer.SetElement (Xamarin.Forms.VisualElement element) [0x000a5] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.458 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.Platform.CreateRenderer (Xamarin.Forms.VisualElement element) [0x0001f] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.458 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.VisualElementPackager.AddChild (Xamarin.Forms.VisualElement view, Xamarin.Forms.Platform.Android.IVisualElementRenderer oldRenderer, Xamarin.Forms.Platform.Android.RendererPool pool, System.Boolean sameChildren) [0x00023] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.458 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.VisualElementPackager.SetElement (Xamarin.Forms.VisualElement oldElement, Xamarin.Forms.VisualElement newElement) [0x0010d] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.458 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.VisualElementPackager.Load () [0x0000d] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.458 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].SetPackager (Xamarin.Forms.Platform.Android.VisualElementPackager packager) [0x00007] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.458 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].SetElement (TElement element) [0x000d5] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.458 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].Xamarin.Forms.Platform.Android.IVisualElementRenderer.SetElement (Xamarin.Forms.VisualElement element) [0x00027] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.458 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.Platform.CreateRenderer (Xamarin.Forms.VisualElement element) [0x0001f] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.458 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.NavigationRenderer.SwitchContentAsync (Xamarin.Forms.Page view, System.Boolean animated, System.Boolean removed) [0x00028] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.458 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.NavigationRenderer.OnPushAsync (Xamarin.Forms.Page view, System.Boolean animated) [0x00000] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.458 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.NavigationRenderer.PushViewAsync (Xamarin.Forms.Page page, System.Boolean animated) [0x00000] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.458 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.NavigationRenderer.OnElementChanged (Xamarin.Forms.Platform.Android.ElementChangedEventArgs`1[TElement] e) [0x000e8] in <766ec173b1324c86bd56e56a733f28e8>:0 
09-27 09:38:57.458 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].SetElement (TElement element) [0x000b9] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.458 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.VisualElementRenderer`1[TElement].Xamarin.Forms.Platform.Android.IVisualElementRenderer.SetElement (Xamarin.Forms.VisualElement element) [0x00027] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.458 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.Platform.CreateRenderer (Xamarin.Forms.VisualElement element) [0x0001f] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.458 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.Platform.AddChild (Xamarin.Forms.VisualElement view, System.Boolean layout) [0x00015] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.459 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.Platform.SetPage (Xamarin.Forms.Page newRoot) [0x00081] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.459 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.FormsApplicationActivity.InternalSetPage (Xamarin.Forms.Page page) [0x000b7] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.459 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.FormsApplicationActivity.SetMainPage () [0x0000c] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.459 21876 21876 I MonoDroid:   at Xamarin.Forms.Platform.Android.FormsApplicationActivity.LoadApplication (Xamarin.Forms.Application application) [0x00045] in <766ec173b1324c86bd56e56a733f28e8>:0
09-27 09:38:57.459 21876 21876 I MonoDroid:   at MyApp.Android.MainActivity.OnCreate (Android.OS.Bundle bundle) [0x0009a] in <a64b72cc8d8641c9b005a4111a2887fc>:0
09-27 09:38:57.459 21876 21876 I MonoDroid:   at Android.App.Activity.n_OnCreate_Landroid_os_Bundle_ (System.IntPtr jnienv, System.IntPtr native__this, System.IntPtr native_savedInstanceState) [0x0000f] in <b7ff78e35d0c4b51b52910be5e614b0f>:0
09-27 09:38:57.459 21876 21876 I MonoDroid:   at (wrapper dynamic-method) System.Object:397b10e2-abad-43bf-a890-b11918008dd9 (intptr,intptr,intptr)
09-27 09:38:57.490 21876 21876 W art     : JNI RegisterNativeMethods: attempt to register 0 native methods for android.runtime.JavaProxyThrowable
09-27 09:38:57.612 31036  8825 D GraphicsStats: Buffer count: 8
09-27 09:38:57.612 31036 31697 I WindowManager: WIN DEATH: Window{d11d9ec u0 com.mycompany.MyApp/md5c79fa08bcb7b9bf8fc5b2b2290dec874.MainActivity}
09-27 09:38:57.614 31036  8825 I ActivityManager: Process com.mycompany.MyApp (pid 21876) has died
```
Please look at **09-27 09:38:57.458 21876 21876 I MonoDroid:   at FFImageLoading.Forms.Droid.CachedImageRenderer.OnElementChanged (Xamarin.Forms.Platform.Android.ElementChangedEventArgs`1[TElement] e) [0x000dd]**

I'll try to revert FFImageLoading, to be sure, that 2.2.16 is affected.

**Comment by daniel-luberda** [comment]
Fixed in 2.2.17, somehow I forgot to push one commit. Sorry for that.

**Comment by Szymaniuk** [comment]
Daaaamn, you're so fast :D. Thank's a lot!

PS: Masz piwo u mnie za zajebistą robotę 🥇

**Comment by michelmoorlag** [comment]
I was about to file an issue for this but it is already fixed. This deservers a 🥇

**Comment by Szymaniuk** [comment]
Confirmed, fixed in 2.2.17 👍
